apps/frontend/app/(client)/(code-editor)/_components/ContestProblemDropdown.tsx

```diff
@@ -0,0 +1,76 @@
+'use client'
+
+import {
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger
+} from '@/components/shadcn/dropdown-menu'
+import { cn, convertToLetter, fetcherWithAuth } from '@/libs/utils'
+import checkIcon from '@/public/icons/check-green.svg'
+import type { ContestProblem, ProblemDetail } from '@/types/type'
+import { useQuery } from '@tanstack/react-query'
+import type { Route } from 'next'
+import Image from 'next/image'
+import Link from 'next/link'
+import { FaSortDown } from 'react-icons/fa'
+
+interface ContestProblemProps {
```

Comment:
```suggestion
interface ContestProblemsResponse {
```

API 응답에 해당하는 타입이니까 위처럼 타입 이름 수정하면 좋을 것 같아요!